core-services/prow/02_config/generate-boskos.py

```diff
@@ -43,6 +43,14 @@
         'us-east-1': 25,
         'ap-northeast-1': 5,
     },
+    'aws-terraform-qe-quota-slice': {
+        'ap-northeast-1': 3,
+        'ap-southeast-1': 3,
+        'us-east-1': 3,
+        'us-east-2': 3,
+        'us-west-1': 3,
+        'us-west-2': 3,
```

Comment:
@amalykhi This number means the concurrent jobs number, I'd suggest starting at 3 or 5, we can increase this value if quota issue appears

Reply:
Done